### PR TITLE
Update code-scanning.md

### PR DESCRIPTION
**Pull request by cmboling**
Some improvements to be added during a 1:1 sync @knewbury01 💯

### CODE REVIEW

File: exercises/code-scanning.md [hunk 1]
```diff
@@ -162,36 +162,67 @@ By creating our own [code scanning suite](https://codeql.github.com/docs/codeql-
...
+
+**Note**: If you want to use these additional query suites and the custom query suite you've made, make sure to import the proper query packs to continue to exclude certain queries.
+
 <details>
 <summary>Solution</summary>
 
 ```yaml
 # Reusing existing QL Pack
-- import: codeql-suites/javascript-code-scanning.qls
+- import: codeql-suites/javascript-security-and-quality.qls
```
Comment (knewbury01) [inline exercises/code-scanning.md:194]: I think this change should also be made on line 152 ect as well.

**Review by knewbury01 — COMMENTED** [review]
would like to add the 2 extra spaces on front of line 161 as well (in front of `id:`) 
I think that one might be tricky for people to figure out on their own later as to why it wont work as is... (might just be me XD)

File: exercises/code-scanning.md (hunk 1 above)
Comment (cmboling) [reply exercises/code-scanning.md:194]: Oooh let's retain it actually since that at point of the exercise we haven't dealt with using additional queries

Comment (cmboling) [reply exercises/code-scanning.md:194]: Just added those 2 extra spaces! Good catch!

Comment (knewbury01) [reply exercises/code-scanning.md:194]: ahh right right ! my bad!

ok cool everything lgtm 👍

**Pull request merged**